servicetalk-examples/http/debugging/src/main/java/io/servicetalk/examples/http/debugging/DebuggingExampleClient.java

```diff
@@ -120,31 +126,44 @@
  * 2021-03-26 19:42:07,381 servicetalk-global-io-executor-1-2 [TRACE] servicetalk-examples-wire-logger - [id: 0xf11baf09, L:/127.0.0.1:57266 - R:localhost/127.0.0.1:8080] FLUSH
  * 2021-03-26 19:42:07,381 servicetalk-global-io-executor-1-2 [TRACE] servicetalk-examples-wire-logger - [id: 0xf11baf09, L:/127.0.0.1:57266 - R:localhost/127.0.0.1:8080] FLUSH
  * 2021-03-26 19:42:07,381 servicetalk-global-io-executor-1-2 [TRACE] servicetalk-examples-wire-logger - [id: 0xf11baf09, L:/127.0.0.1:57266 - R:localhost/127.0.0.1:8080] READ_REQUEST
+ * 2021-03-26 19:42:07,394 servicetalk-global-io-executor-1-2 [TRACE] servicetalk-examples-wire-logger - [id: 0xf11baf09, L:/127.0.0.1:57266 - R:localhost/127.0.0.1:8080] CLOSE
+ * 2021-03-26 19:42:07,394 servicetalk-global-io-executor-1-2 [TRACE] servicetalk-examples-wire-logger - [id: 0xf11baf09, L:/127.0.0.1:57266 ! R:localhost/127.0.0.1:8080] INACTIVE
+ * 2021-03-26 19:42:07,395 servicetalk-global-io-executor-1-2 [TRACE] servicetalk-examples-wire-logger - [id: 0xf11baf09, L:/127.0.0.1:57266 ! R:localhost/127.0.0.1:8080] UNREGISTERED
  * </pre>
  */
 public final class DebuggingExampleClient {
 
     static {
-        // 1. (optional) Disables the AsyncContext associated with individual request/responses to reduce stack-trace
-        // depth and simplify execution tracing. This will disable/break some features such as request tracing,
-        // authentication, propagated timeouts, etc. that rely upon the Async context so should only be disabled when
-        // necessary for debugging
-        /* AsyncContext.disable(); */
+        /*
+         * 1. (optional) Disables the AsyncContext associated with individual request/responses to reduce stack-trace
+         * depth and simplify execution tracing. This will disable/break some features such as request tracing,
+         * authentication, propagated timeouts, etc. that rely upon the AsyncContext so should only be disabled when
+         * necessary for debugging:
+         */
+        // AsyncContext.disable();
```

Comment:
The main motivation for the flipped change here was that Intellij IDEA adds `//` when I tried to use a shortcut to uncomment `/* AsyncContext.disable(); */`.